src/components/BottomSheet/BottomSheetMenu.js

```diff
@@ -53,15 +53,15 @@ const styles = StyleSheet.create({
     borderBottomWidth: 1,
     paddingVertical: dimen.normalizeDimen(14),
     paddingHorizontal: dimen.normalizeDimen(20),
-    borderColor: colors.greySubtile2
+    borderColor: COLORS.greySubtile2
   },
   rbsheetContent2: {
     flexDirection: 'row',
     alignItems: 'center',
     borderBottomWidth: 1,
     paddingVertical: dimen.normalizeDimen(14),
     paddingHorizontal: dimen.normalizeDimen(20),
-    borderColor: colors.greySubtile2
+    borderColor: COLORS.greySubtile2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `borderColor` property in the `rbsheetContent2` style has been updated to use the new `COLORS` object. This change is consistent with the PR's goal of standardizing color usage across the application.
```diff
-    borderColor: colors.greySubtile2
+    borderColor: COLORS.greySubtile2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->